WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -273,11 +282,15 @@ protected void onPause() {
 
     @Override
     protected void onSaveInstanceState(Bundle outState) {
-        outState.putInt(SAVED_NAV_POSITION, mNavPosition);
         outState.putInt(SAVED_WP_LOGIN_STATE, mResultCode);
         outState.putInt(ARG_LOCAL_TABLE_BLOG_ID, mLocalBlogID);
         outState.putSerializable(SAVED_STATS_TIMEFRAME, mCurrentTimeframe);
         outState.putString(SAVED_STATS_REQUESTED_DATE, mRequestedDate);
+        outState.putIntArray(SAVED_STATS_SCROLL_POSITION,
+                new int[]{
+                        mOuterScrollView.getScrollX(),
```

Comment:
It's not 100% accurate because the country module changes dimensions upon rotation. All others modules have the same height. 
I think that for now the current implementation that uses the Y scrolling offset is fine. We could change it later if/when the module changes dimensions/layout in landscape.